docker/Dockerfile.server.alpine.multiarch

```diff
@@ -6,7 +6,9 @@ ENV CA_CERTIFICATES_VERSION="20230506-r0"
 ARG TARGETOS TARGETARCH
 RUN apk add -U --no-cache ca-certificates=${CA_CERTIFICATES_VERSION}
 ENV GODEBUG=netdns=go
-ENV WOODPECKER_DATABASE_DATASOURCE=/var/lib/woodpecker/woodpecker.sqlite
+# create datasource config file
+RUN echo -n "/var/lib/woodpecker/woodpecker.sqlite" > /etc/woodpecker-datasource.conf
```

Comment:
1. Looks like a garbage file in case of using non-defaults (postgres, mysql)
2. There is nothing to hide in case of sqlite
3. A little inconsistency with [/etc/woodpecker/agent.conf](https://github.com/woodpecker-ci/woodpecker/blob/4e44dd0e76b516685bc79af4885bcfb739720986/cmd/agent/core/flags.go#L58)